components/server/src/authorization/relationship-updater.ts

```diff
@@ -52,13 +66,7 @@ export class RelationshipUpdater {
             fromVersion: user?.additionalData?.fgaRelationshipsVersion,
             toVersion: this.version,
         });
-        const orgs = await this.orgDB.findTeamsByUser(user.id);
```

Comment:
Just for my understanding:

Is it because updateUser and updateOrganization can reconcile properly? It cannot be that we are missing an org if a user was removed out of it because then relationship will be already updaed?

Reply:
Not entirely sure I understand the question, but try to answer anyway. Let me know if that does not answer it.

We are updating all of the user's orgs as well as those orgs where we find relationships to the given user, because we want the relationship updater to correctly sync the DB state for a given user to the spicedb relationship state. That implies we need to get rid of any stale relationships. We cannot just wipe everything because some relationships (e.g. between org and project) may be used concurrently by other users.